src/main/java/Controllers/AppointmentServlet.java

```diff
@@ -7,6 +7,8 @@
 
 import Models.Appointment.Appointment;
 import Models.Appointment.AppointmentDAO;
+import Models.Prescription.Prescription;
```

Comment:
Correct me if I'm wrong.

Is the plan to revert changes to AppointmentServlet.java and Appointments.jsp?

Reply:
We won't be needing them as they will be deleted in the fat controllers branch